core/test/loader/atom_media.test.ts

```diff
@@ -0,0 +1,56 @@
+import '../dom-parser.js'
```

Comment:
Let’s put it to the `atom.test.ts` if we do not have a strong reasons to move this specific case